code/game/objects/items/devices/radio/headset.dm

```diff
@@ -185,14 +185,15 @@
 
 /obj/item/device/radio/headset/heads/hos
 	name = "warrant officer headset"
-	desc = "The headset of the men who lock away your worthless lives. This has a small symbol denoting its built in back-up transmitter."
+	desc = "The headset of the men who locks away your worthless lives. This has a small symbol denoting its built in back-up transmitter."
```

Comment:
this was grammatical before; "lock" is referring to "the men"

Reply:
I thought it was refering to the "men" as the plural of the WO that <verb>, I see